frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -458,7 +458,9 @@ const DataTable = ({
                 <th
                   className={"active-selection__checkbox"}
                   {...headerGroups[0].headers[0].getHeaderProps(
-                    headerGroups[0].headers[0].getSortByToggleProps()
+                    headerGroups[0].headers[0].getSortByToggleProps({
+                      title: null,
+                    })
```

Comment:
Here we eliminate the default React `Toggle SortBy` tooltip as a partial fix for https://github.com/fleetdm/fleet/issues/15782